Sources/Kitura/Router.swift

```diff
@@ -92,8 +92,8 @@ public class Router {
     ///                                FileResourceServer to serve the default
     ///                                "Welcome to Kitura" page and related
     ///                                assets.
-    public init(mergeParameters: Bool = false, enableWelcomePage: Bool = true) {
-        self.swagger = SwaggerDocument()
+    public init(mergeParameters: Bool = false, enableWelcomePage: Bool = true, apiDocument: SwaggerDocument = SwaggerDocument()) {
```

Comment:
This looks good, can you add the documentation comment for the new `apiDocument` parameter? Something like
> - Parameter apiDocument: Optional parameter that allows customization of the OpenAPI document describing this Router.